<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="mainWindowBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>571</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newpic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>571</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New pic</string>
   </property>
  </widget>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
